Products/PortalTransforms/tests/test_transforms.py

```diff
@@ -201,6 +201,20 @@ def setUp(self):
         registry = getUtility(IRegistry)
         self.settings = registry.forInterface(
             IFilterSchema, prefix="plone")
+        self.settings.valid_tags.append('style')
+        import transaction
+        transaction.commit()
```

Comment:
@MrTango if you make manual commits you need to switch from integration to functional layer, if not we will have test isolation problems later on